ui/src/shared/constants/fluxFunctions.ts

```diff
@@ -119,36 +123,28 @@ export const FLUX_FUNCTIONS: FluxToolbarFunction[] = [
         type: 'Boolean',
       },
     ],
+    package: '',
     desc: 'Applies an aggregate function to fixed windows of time.',
     example: 'aggregateWindow(every: 1m, fn: mean)',
     category: 'Aggregates',
     link:
-      'https://v2.docs.influxdata.com/v2.0/reference/flux/functions/transformations/aggregates/aggregatewindow',
+      'https://v2.docs.influxdata.com/v2.0/reference/flux/functions/built-in/transformations/aggregates/aggregatewindow/',
   },
   {
-    name: 'assertEquals',
```

Comment:
This seems like a strange update. did they really remove the assertEquals function? it's still in the spec: https://github.com/influxdata/flux/blob/master/docs/SPEC.md#assertequals

Reply:
They didn't remove it. It got moved into the `testing` package.